pyke/lightcurve.py

```diff
@@ -261,9 +294,57 @@ def header(self, ext=0):
 
     def _flux_types(self):
         """Returns a list of available flux types for this light curve file"""
-        return [n for n in  self.hdu[1].data.columns.names if 'FLUX' in n]
+        types = [n for n in self.hdu[1].data.columns.names if 'FLUX' in n]
+        types = [n for n in types if not ('ERR' in n)]
+        return types
 
 
+    def plot(self, plottype = None, ax=None, norm=True, **kwargs):
+        """
+        Plot a Light Curve.
+
+        Parameters
+        ----------
+        plottype : list of FLUX types to plot
+        """
+        if ax is None:
+            fig, ax = plt.subplots(1)
+        if plottype is None:
+            plottype = self._flux_types()
+        if isinstance(plottype,str):
+            plottype=[plottype]
+        if not hasattr(plottype,'__iter__'):
+            plottype=[plottype]
+
+        id = self.SAP_FLUX.keplerid
+        for pl in plottype:
+            try:
+                lc = self.get_lightcurve(pl)
+            except:
+                continue
+            f = lc.flux
+            ferr = lc.flux_err
+            t = self.time
+            if norm:
+                ferr/=np.nanmedian(f)
+                f/=np.nanmedian(f)
+            t0=float(int(t[0] / 100) * 100.0)
+            plt.errorbar(t-t0, f, ferr,label='{}'.format(pl),**kwargs)
+
+
+        plt.legend()
+#        plt.fill(t-t0, f, fc='#a8a7a7', linewidth=0.0, alpha=0.3)
+
+        xlab = 'BJD $-$ {:d}'.format(int(t0+2454833))
+        if norm:
+            ylab1 = 'Normalized Flux'
+        else:
+            ylab1 = 'Flux (e$^-$ s$^{-1}$)'
+        plt.xlabel(xlab, {'color' : 'k'})
+        plt.ylabel(ylab1, {'color' : 'k'})
+        plt.title('Kepler ID: {}'.format(id))
+        plt.grid()
+
```

Comment:
I think this plot function could go in ``utils.py`` just like ``plot_image``. This way, we avoid code replication.